packages/odf/components/odf-dashboard/status-card/status-card.tsx

```diff
@@ -129,6 +138,23 @@ export const StatusCard: React.FC = () => {
     csvLoadError
   );
 
+  const isProviderMode = useFlag(PROVIDER_MODE);
+
+  const [clients, clientsLoaded, clientsLoadError] = useK8sWatchResource<
+    StorageConsumerKind[]
+  >({
+    kind: referenceForModel(StorageConsumerModel),
+    isList: true,
+  });
+
+  const clientAggregateHealth = getAggregateClientHealthState(clients);
```

Comment:
should we for safety to prevent type error ?? or `useK8sWatchResource` will guaranteed return empty list meanwhile when resources are loading or if in error ??
```suggestion
  const clientAggregateHealth = getAggregateClientHealthState(clients || []);
```

Reply:
or maybe we can directly do `state={getAggregateClientHealthState(clients)}`, because then this function will only be called when request is loaded with no error...

Reply:
I will add a default value in the function parameter.